recipe/meta.yaml

```diff
@@ -40,6 +40,7 @@ requirements:
 #    - cuda-version >={{ cuda_compiler_version }}  # [cuda_compiler_version not in (undefined, 'None')]
     - make
     - cmake
+    - gxx <12
```

Comment:
Please don't do this. If you need a specific compiler version (or avoid another), do the following in `conda_build_config.yaml`:
```
# comment for context
c_compiler_version:            # [linux]
  - 11                         # [linux]
cxx_compiler_version:          # [linux]
  - 11                         # [linux]
```

Reply:
Got it, I was doing this via the cuda120.yaml migrator, which tries to bring gcc 12 and causes issues with pybind11. Is yours a better solution in general?

Reply:
Adding this file appears to not be enough, gcc 12.3 is still installed
```shell
    gcc_impl_linux-64:                    12.3.0-he2b93b0_3       conda-forge
    gcc_linux-64:                         12.3.0-h76fc315_2       conda-forge
    gxx_impl_linux-64:                    12.3.0-he2b93b0_3       conda-forge
    gxx_linux-64:                         12.3.0-h8a814eb_2       conda-forge
```